doc/workshop/ExternalModels/projectile.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#***************************************
+#* Simple analytic test ExternalModule *
+#***************************************
+#
+# Simulates time-dependent track of a projectile through the air from start to 0,
+#     assuming no air resistance.
+#     Inputs:
+#       (x0,y0) - initial position
+#       v0 - initial total velocity
+#       ang - angle of initial motion, in degrees, with respect to flat ground
+#     Outputs:
+#       (x,y) - vector positions of projectile in time
+#       t - corresponding time steps
+#
```

Comment:
Can this file be merged with the one in AnalyticModels?

Reply:
agree

Reply:
It can be merged, but I would like to keep a copy at this Folder just for the convenient.